.vitepress/theme/components/Page.vue

```diff
@@ -19,7 +19,7 @@ import PageFooter from './PageFooter.vue'
 import NextAndPrevLinks from './NextAndPrevLinks.vue'
 </script>
 
-<style scoped>
+<style scoped lang="scss">
```

Comment:
You can remove this, also delete scss package.

Reply:
Is there a reason not to use `scss`? It's much easier to write

Reply:
Removed